voidcrew/area/ship_areas.dm

```diff
@@ -0,0 +1,7 @@
+/area/asteroid
```

Comment:
This area was repathed to ``/area/centcom/asteroid``, we don't need to add it it still exists